etc/grafana/dashboards/overview.json

```diff
@@ -4596,7 +4584,7 @@
             "uid": "${DS_PROMETHEUS}"
           },
           "editorMode": "builder",
-          "expr": "increase(reth_network_pool_transactions_messages_sent{instance=~\"$instance\"}[$__rate_interval])",
+          "expr": "rate(reth_network_pool_transactions_messages_sent{instance=~\"$instance\"}[$__rate_interval])",
```

Comment:
the unit here, `$__rate_interval` is dynamic. but the units `reqps/s` is fixed. same was happening for `InsertedTransactions`, see https://github.com/paradigmxyz/reth/pull/6646 on how to fix it.

Reply:
Thanks for the advice, but I want to use `rate` to represent the speed of pool_txs sent/received per second, so i think the magic `$__rate_interval` is working for `rate`, but not working for `increase`, because the later require the interval of time

Reply:
or we can calculate the `rate` with fixed interval, eg: `1m`, `5m`

Reply:
@jsvisa maybe you can figure out how to do the expression `${query} / 60` as a transformation instead of an expression? that would be relevant for the linked pr too.

Reply:
I see, but the interval in `rate` and `increase` are different, the two function defined as:

From https://prometheus.io/docs/prometheus/latest/querying/functions/#rate

> rate(v range-vector) calculates the per-second average rate of increase of the time series in the range vector.

And https://prometheus.io/docs/prometheus/latest/querying/functions/#increase

> increase(v range-vector) calculates the increase in the time series in the range vector. 

so `rate([1m])` is similar to `rate([2m])`, the differ is the later will collect more data(time-series in 2m vs in 1m) to calculate the per-second increase. In the meantime, `incrase([1m])` is different to `increase([2m])`, about 2 times bigger. 



> magic variable

<img width="2145" alt="image" src="https://github.com/paradigmxyz/reth/assets/3627395/68fc71f9-7052-476f-8485-7b01212f1b3a">

> 1m

<img width="2161" alt="image" src="https://github.com/paradigmxyz/reth/assets/3627395/d8e37cbd-bca9-4a18-9759-e70bec8d221a">

> 5m

<img width="2138" alt="image" src="https://github.com/paradigmxyz/reth/assets/3627395/299ab6f8-a0e4-4aae-bd1f-c19bfbcab53c">


> 10m 

<img width="2139" alt="image" src="https://github.com/paradigmxyz/reth/assets/3627395/bf8efc29-d2b3-4956-adab-1bd8992e0298">


As you can see, the result is similar, and more smoother if the interval is larger


Ref: https://grafana.com/blog/2020/09/28/new-in-grafana-7.2-__rate_interval-for-prometheus-rate-queries-that-just-work/

Reply:
ok! thanks for the verbose and referenced explanation, appreciate that. then the linked pr, could also have been fixed by using rate instead of increase. will commit that fix right away.